### ISSUE TITLE
Can someone add Asus Strix Scar 2 gl504gm config? pleasee

### ISSUE THREAD

**Issue by yokaiemporer** [hirschmann/nbfc#1064]
I tried a lot to build a config but the ec-probe.exe doesn't give any output,it just prints 00 .
chaning values in r/w anything doesnt seem to reflect anything as well .
I have the dsdt tables if anyone needs .. please include this config

**Comment by alone-86** [comment]
I've tried many, many things that didn't quite fix the problem. My computer is the same model as yours and I know the goal is to get the damn fan to stop going crazy for no reason. For what I know so far, the only effective method is to unplug the faninterface again. This method does not permanently solve the fan problem.

For nbfc, I searched a lot about EC in order to create the profile and learned how it works with the south bridge and BIOS. But using RW or ec-probe.exe reads nothing but the damn 00.

I'm in China and the authorized service providers will just stupidly reinstall the OS and reinstall the drivers. They said they also replaced the motherboard, but this problem will still come back after some time.

About the method of unplugging the fan I mentioned https://rog.asus.com/forum/showthread.php?104242-Strix-SCAR-II-GL504GS-Fan-Question-Stuck-on-High-for-a-while

i try so hard and get so far, but in the end, it doesn't matter....
